packages/perseus/src/strings.ts

```diff
@@ -321,7 +321,7 @@ export const strings: {
     graphKeyboardPrompt: "Press Shift + Enter to interact with the graph",
     srPointAtCoordinates: {
         context: "Screenreader-accessible description of a point on a graph",
-        message: "Point %(pointNum) at %(x)s comma %(y)s",
+        message: "Point %(num)s at %(x)s comma %(y)s",
```

Comment:
For posterity, the issue was the lack of trailing `s` and not the fact that the value was named `num`. :)